src/module/actor/npc/sheet.ts

```diff
@@ -161,12 +162,36 @@ class NPCSheetPF2e<TActor extends NPCPF2e> extends CreatureSheetPF2e<TActor> {
         level.adjustedHigher = level.value > Number(level.base);
         level.adjustedLower = level.value < Number(level.base);
         const { ac, hp, perception, hardness } = sheetData.data.attributes;
+        const speedData = sheetData.data.attributes.speed;
         ac.adjustedHigher = ac.value > Number(ac.base);
         ac.adjustedLower = ac.value < Number(ac.base);
         hp.adjustedHigher = hp.max > Number(hp.base);
         hp.adjustedLower = hp.max < Number(hp.base);
         perception.adjustedHigher = perception.totalModifier > Number(perception.base);
         perception.adjustedLower = perception.totalModifier < Number(perception.base);
+        sheetData.speeds = {
+            land: {
+                label: speedData.label ?? "",
+                value: speedData.value,
+                details: speedData.details,
+                adjustedHigher: speedData.total > this.actor._source.system.attributes.speed.value,
+                adjustedLower: speedData.total < this.actor._source.system.attributes.speed.value,
```

Comment:
This happens so often that I think we should make this a sheet util. I kinda made a util in the vehicle sheet that could probably be fleshed out more for such things.

Reply:
This is dissimilar to all other "adjustable" fields.

Reply:
If by this you mean the vehicle, its the same, I just replaced the adjustedHigher/adjustedHigher if/else in the sheets with a single adjusted css class we can mindless throw in. If by this you mean this npc sheet thing, maybe, but easily rectifiable with a splat operator.

That said if we clean up all that adjustedHigher/lower stuff they should likely be done all at once anyways.